kratos/python_scripts/read_materials_process.py

```diff
@@ -64,32 +64,24 @@ def _GetVariable(self,my_string):
 
         return KratosMultiphysics.KratosGlobals.GetVariable(splitted[-1]) # This also checks if the application has been imported
 
-    def _GetConstitutiveLaw(self,my_string):
+    def _GetConstitutiveLaw(self, my_string):
         """Return the python object of a Constitutive Law named by the string argument.
 
         Example:
         constitutive_law = self._GetConstitutiveLaw('KratosMultiphysics.StructuralMechanicsApplication.LinearElastic3DLaw')
+        constitutive_law = self._GetConstitutiveLaw('StructuralMechanicsApplication.LinearElastic3DLaw')
```

Comment:
will this format still work?

Reply:
You can use anything before the `.NameOfLaw`, it simply ignores what is before
So e.g. also `SomeDefault.Whatever.LinearElastic3DLaw` works

you can use max two arguments before the lawname, to accomodate `KratosMultiphysics.MyApplication. ...`

Reply:
Mmm...ignoring it silently is not the best option. we could

Option A - use "what is before" to see if the app is imported
Option B - throw a warning telling that the text is being ignored

Reply:
The new macro internally if the application is imported (i.e. if it is not imported, the const law is not registered in Kratos::Components and will [throw an error saying exactly this](https://github.com/KratosMultiphysics/Kratos/blob/b1d0d7d98c9d23826a11e423956ec4ca79b616e3/kratos/python_interface/kratos_globals.py#L91))

Therefore +1 for throwing a warning

Reply:
Done